docs/sources/configuration/_index.md

```diff
@@ -3939,6 +3943,25 @@ embedded_cache:
   # CLI flag: -<prefix>.embedded-cache.ttl
   [ttl: <duration> | default = 1h]
 
+lru_cache:
+  # Maximum memory size of the whole cache. A unit suffix (KB, MB, GB) may be
+  # applied. Default: 1GB.
+  # CLI flag: -<prefix>.lrucache.max-size-bytes
+  [max_size_bytes: <int> | default = 0B]
+
+  # Maximum memory size of a single item in the cache. A unit suffix (KB, MB,
+  # GB) may be applied. Default: 100MB.
+  # CLI flag: -<prefix>.lrucache.max-item-size-bytes
+  [max_item_size_bytes: <int> | default = 0B]
```

Comment:
Is the default 100BM (line 3953) or 0B (line 3955) ?

Reply:
It should be 100MB but the `make docs` task is incorrectly setting this to 0b :(